lib/lbt/resources/ResourceCollector.js

```diff
@@ -253,19 +272,17 @@ class ResourceCollector {
 
 	groupResourcesByComponents(options) {
```

Comment:
`options` could be removed completely, right?

Reply:
Yes 🙌